VaticanBAV/et/BAVet117.xml

```diff
@@ -0,0 +1,1158 @@
+                              </item>
+                              <item xml:id="q4" n="3">
+                                <dim unit="leaf">8</dim>
+                                <locus from="25r" to="32v"></locus>
+                              </item>
+                              <item xml:id="q5" n="4">
+                                <dim unit="leaf">10</dim>
+                                <locus from="33r" to="42v"></locus>
+                              </item>
+                              <item xml:id="q6" n="5">
+                                <dim unit="leaf">10</dim>
+                                <locus from="43r" to="52v"></locus>
+                              </item>
+                              <item xml:id="q7" n="6">
+                                <dim unit="leaf">10</dim>
+                                <locus from="53r" to="62v"></locus>
+                              </item>
+                              <item xml:id="q8" n="7">
+                                <dim unit="leaf">10</dim>
+                                <locus from="63r" to="72v"></locus>
+                              </item>
+                              <item xml:id="q9" n="8">
+                                <dim unit="leaf">10</dim>
+                                <locus from="73r" to="82v"></locus>
+                              </item>
+                              <item xml:id="q10" n="9">
+                                <dim unit="leaf">10</dim>
+                                <locus from="83r" to="92v"></locus>
+                              </item>
+                            </list>
+                          </collation>
+                        </supportDesc>
+                        <layoutDesc>
+                          <layout columns="1" writtenLines="19"/>
+                        </layoutDesc>
+                      </objectDesc>
+                      <handDesc>
+                        <handNote xml:id="h1" script="Ethiopic">
+                          <seg type="rubrication">
+                            Titles
+                          </seg>
+                        </handNote>
+                      </handDesc>
+                      <additions>
+                        <list>
+                          <item xml:id="a1">
+                            <locus from="1r" to="4r"/>
+                            <desc>The scribe began to copy <title type="complete" ref="LIT1295Dersan"/>, leaving space to paint an image of him.</desc>
+                          </item>
+                          <item xml:id="a2">
+                            <locus from="1v" to="4v"/>
+                            <desc>Pen trials.</desc>
+                          </item>
+                          <item xml:id="a3">
+                            <locus target="#63r"/>
+                            <desc>A ፫ in the interior margin, used to number the quire, indicates that the manuscript was formed from two manuscripts.</desc>
```

Comment:
Would also go with 2 msParts: we have new quire numbering from f. 43 and a new text. 
The problem is, we have no hint (I guess?) on a change in layout or similar, so it could be still two parts of the same production process (2 production units but just one 1 circulation unit), which is a border case, but still we have 3 coinciding breaks (quire / quire sequence / text unit) that are a strong case for 2 msParts